src/test/ui/suggestions/missing-comma-in-match.stderr

```diff
@@ -0,0 +1,12 @@
+error: expected one of `,`, `.`, `?`, `}`, or an operator, found `=>`
+  --> $DIR/missing-comma-in-match.rs:14:18
+   |
+13 |         &None => 1
+   |               --  - help: missing a comma here to end this match arm
+   |               |
+   |               while parsing the match arm starting here
```

Comment:
I feel like this is "heavy", but I'm not sure how to suggest an improvement yet. I wonder if just the suggestion would be enough? 

```
error: expected one of `,`, `.`, `?`, `}`, or an operator, found `=>`
  --> $DIR/missing-comma-in-match.rs:14:18
   |
13 |         &None => 1
   |                   - help: maybe missing a comma here: `1,`
14 |         &Some(2) => { 3 }
   |                  ^^ expected one of `,`, `.`, `?`, `}`, or an operator here

error: aborting due to previous error
```

Would that be hard to add?

Reply:
It shouldn't.